apps/browser/src/autofill/notification/bar.ts

```diff
@@ -30,7 +30,16 @@ function load() {
     notificationChangeDesc: chrome.i18n.getMessage("notificationChangeDesc"),
   };
 
-  document.getElementById("logo-link").title = i18n.appName;
+  const logoLink = document.getElementById("logo-link") as HTMLAnchorElement;
+  logoLink.title = i18n.appName;
+
+  // Update logo link to user's regional domain
+  const webVaultURL = getQueryVariable("webVaultURL");
+  const newVaultURL = webVaultURL && decodeURIComponent(webVaultURL);
+
+  if (newVaultURL && newVaultURL !== logoLink.href) {
```

Comment:
```suggestion
  if (newVaultURL !== logoLink.href) {
```
Nit - unless we're expecting `logoLink.href` to possibly be a falsy value, it should be safe to remove the truthy check on `newVaultURL`.

Not blocking because of this though.